pkgs/by-name/qu/quickemu/package.nix

```diff
@@ -73,6 +74,7 @@ stdenv.mkDerivation (finalAttrs: {
       -e '/OVMF_CODE_4M.fd/s|ovmfs=(|ovmfs=("${OVMF.firmware}","${OVMF.variables}" |' \
       -e '/cp "''${VARS_IN}" "''${VARS_OUT}"/a chmod +w "''${VARS_OUT}"' \
       -e 's/Icon=.*qemu.svg/Icon=qemu/' \
+      -e 's,\$(command -v smbd),${pkgs.samba}/bin/smbd,' \
```

Comment:
Well I think I failed to make it clear, the whole check can be hardcoded to true, since it's qemu's job to find smbd, not quickemu's.

Reply:
True but if qemu fails to find smbd it will fail to start. Thus it makes sense for quickemu to figure out if qemu supports smbd. In our case we know it does, so yes, the check can be set to true. I've applied your suggestion and rebased the branch.